test/cli/more-projects_test.py

```diff
@@ -636,4 +636,11 @@ def test_json_file_ignore_2(tmpdir):
         'cppcheck: all paths were ignored'
     ]
 
-    assert_cppcheck(args, ec_exp=1, err_exp=[], out_exp=out_lines)
\ No newline at end of file
+    assert_cppcheck(args, ec_exp=1, err_exp=[], out_exp=out_lines)
+
+
+def test_project_missing_files():
+    filename = os.path.join('invalid-project', 'main.c')
+    ret, _, stderr = cppcheck(['--template=cppcheck1', '--project=' + os.path.join('invalid-project', 'invalid-project.vcxproj')])
```

Comment:
it looks reasonable to run cppcheck and then just check exit code and stderr output. :+1: